Using Row panel for CPU and IO Queue information

amnonh opened the pull request:
This series makes use of Grafana Row-Panel to combine information in a templated row (multiple instances of the same row with different parameter values).
The CPU dashboard, has now row per task-group, with utilization, quota violation and share.

The IO queue information is now collected in rows.
![image](https://user-images.githubusercontent.com/2118079/77880333-15ba0f80-7265-11ea-9edd-142120cfd27a.png)
![image](https://user-images.githubusercontent.com/2118079/77880347-1e124a80-7265-11ea-9b37-b14506f92919.png)

Fixes #857
Fixes #846

amnonh committed:
3.2: use panel row to show cpu and io information

amnonh committed:
3.3: use panel row to show cpu and io information

amnonh committed:
master: use panel row to show cpu and io information

amnonh committed:
3.1: use panel row to show cpu and io information

amnonh committed:
add the generated dashboard

glommer commented:
Hi Amnon

Which metric are we using for the foreground cpu time ? 
We'll probably be better off ditching that graph because it's pretty much always wrong now for similar reasons: if you use workload prioritization, your foreground classes have different names

amnonh commented:
@glommer scylla_scheduler_runtime_ms{group="main"}+scylla_scheduler_runtime_ms{group="statement"}

glommer commented:
This is wrong. The class is not called statement if Workload Prioritization
is used.

Let's just remove it. With all the classes down below, it is not important
to group foreground

On Wed., Apr. 1, 2020, 1:12 p.m. Amnon Heiman, <notifications@github.com>
wrote:

> @glommer <https://github.com/glommer>
> scylla_scheduler_runtime_ms{group="main"}+scylla_scheduler_runtime_ms{group="statement"}
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scylladb/scylla-monitoring/pull/869#issuecomment-607377279>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQ3PPR7YEBUNKR5H3CQ53RKNYXFANCNFSM4LWJYMIA>
> .
>

amnonh committed:
scylla-cpu.master: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
scylla-cpu.2019.1: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
scylla-cpu.3.0: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
scylla-cpu.3.3: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
scylla-cpu.3.1: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
scylla-cpu.3.2: remove the cpu utilization from cpu dashboard

Now that we have information per group the "foreground" panels from the
CPU dashboard.

The cpu utilization was removed as it's already part of the over view
dashboard and there's no point in leaving a single panel in a row.

amnonh committed:
add the generated dashboard

amnonh commented:
The updated dashboard looks like:
![image](https://user-images.githubusercontent.com/2118079/78240413-a6a01d80-74e7-11ea-8ddc-24701310e485.png)
